pyproject.toml

```
@@ -0,0 +1,28 @@
+[build-system]
+requires = [ "cmake~=3.24.0", "ninja; platform_system!='Windows'", "setuptools>=42", "toml",]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "PennyLane_Lightning"
+description = "PennyLane-Lightning plugin"
+readme = "README.rst"
+requires-python = ">=3.9"
+classifiers = [ "Development Status :: 4 - Beta", "Environment :: Console", "Intended Audience :: Science/Research", "License :: OSI Approved :: Apache Software License", "Natural Language :: English", "Operating System :: MacOS :: MacOS X", "Operating System :: Microsoft :: Windows", "Operating System :: POSIX", "Operating System :: POSIX :: Linux", "Programming Language :: Python", "Programming Language :: Python :: 3", "Programming Language :: Python :: 3 :: Only", "Programming Language :: Python :: 3.9", "Programming Language :: Python :: 3.10", "Programming Language :: Python :: 3.11", "Programming Language :: Python :: 3.12", "Topic :: Scientific/Engineering :: Physics",]
```

```suggestion
classifiers = [ 
"Development Status :: 4 - Beta", 
"Environment :: Console", 
"Intended Audience :: Science/Research", 
"License :: OSI Approved :: Apache Software License", 
"Natural Language :: English", 
"Operating System :: MacOS :: MacOS X", 
"Operating System :: Microsoft :: Windows", 
"Operating System :: POSIX", 
"Operating System :: POSIX :: Linux", 
"Programming Language :: Python", 
"Programming Language :: Python :: 3", 
"Programming Language :: Python :: 3 :: Only", 
"Programming Language :: Python :: 3.9", 
"Programming Language :: Python :: 3.10", 
"Programming Language :: Python :: 3.11", 
"Programming Language :: Python :: 3.12", 
"Topic :: Scientific/Engineering :: Physics",
]
```

Would formatting like this cause any issues?

---

We don't control formatting. Python/toml will always re-format it when saving the file.